docs/adr/adr-005-email-sending-and-delivery-responsibilities.md

```diff
@@ -0,0 +1,120 @@
+
+## Introduction
+
+The [initial Email Alert API Notify integration][adr-1] was designed to
+monitor whether Notify was delivering emails to recipients and to have
+automatic retry mechanisms when there were problems with this delivery.
+
+In September 2020, we re-evaluated the need for this level of monitoring and
+concluded that this was unnecessary as it duplicated functionality that
+Notify provided. We chose to simplify the system and re-define the
+responsibilities of email sending and delivery. We decided
+that Email Alert API's responsibility in sending email was ensuring a request
+was sent to Notify and that the onward responsibility of email delivery
+was a concern that belonged to Notify.
+
+[adr-1]: adr-001-notify-integration.md
+
+## Specifics
+
+Email, as an asynchronous communication medium, inherently has two boolean
+scenarios that need to be true for an email to be received. An email has to be
+_sent_ to a mail server and the mail server then needs to _deliver_ the email to the
+recipient's mail server. Resolving whether an email is sent is a rather simple
+and question that can be resolved synchronously - "did the mail server accept the
+email?" - however, determining whether an email is received is more complex
+and it can be a drawn out process as mail servers may retry sending over a period
+of time when encountering failures. Typically, in the medium of email, clients
+only consider the first scenario in reporting an email's status, this is the
+point an email becomes sent. Whereas the latter scenario isn't typically
+reflected in email clients, it is normal to assume the email was received
+successfully unless later you receive an email indicating a bounce has occurred.
+
+With regards to Email Alert API we consider the process of sending an email
+to be a successful HTTP request to Notify. We then consider the
+[callback][notify-callback] Notify provides as the mechanism to learn if
+an email was delivered.
+
+We reflected that Email Alert API had diverged from the common pattern of email
+clients by conflating the sending and delivery of email. This resulted
+in the sending of email to be somewhat pessimistic - we considered an email to
+only be sent when it was confirmed to be delivered - and to be
+somewhat confusing, with terminology about sending emails mixed with
+terminology about delivering email.
+
+To resolve these concerns we have switched to considering an email to
+be _sent_ when the request to Notify is successful, which will end the period
+where an email is in a transitory `pending` status. We will consider it the
+responsibility of Notify to manage and monitor the delivery of an email,
+which is already part of the Notify service.
+
+[notify-callback]: https://docs.notifications.service.gov.uk/ruby.html#delivery-receipts
+
+### Consequences
+
+#### Emails will leave their `pending` state when sent to Notify
+
+Email Alert API would consider an email to be in a `pending` status until
```

Comment:
```suggestion
Email Alert API would consider a sent email to be in a `pending` status until
```

This isn't quite true, if we couldn't send it to Notify, or the delivery failed (depending on what period in time we're thinking about).